src/node/waveshaper.rs

```diff
@@ -349,18 +350,19 @@ impl WaveShaperRenderer {
     }
 
     #[inline]
-    fn no_process(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
+    fn pass_through(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
```

Comment:
This could be written as `*output = input.clone();` I think. The clone does not allocate but increases the `Rc` strong count